wp/lib/bap_wp/src/environment.ml

```diff
@@ -37,6 +37,8 @@ module ExprSet = Set.Make(
     let t_of_sexp _ = raise (Not_implemented "t_of_sexp for z3_expr not implemented")
   end)
 
+let (let*) (x : 'a option) (f : 'a -> 'b) = Option.bind x ~f
```

Comment:
We might want to move this within a `Utils.Option_Let` module or something. It feels weird to have it at the top level here.

Reply:
Can do.